client/lxd_cluster.go

```diff
@@ -79,14 +83,14 @@ func (r *ProtocolLXD) GetClusterMemberNames() ([]string, error) {
 
 	// Fetch the raw URL values.
 	urls := []string{}
-	baseURL := "/cluster/members"
-	_, err = r.queryStruct("GET", baseURL, nil, "", &urls)
+	u := api.NewURL().Path("cluster", "members")
+	_, err = r.queryStruct(http.MethodGet, u.String(), nil, "", &urls)
 	if err != nil {
 		return nil, err
 	}
 
 	// Parse it.
-	return urlsToResourceNames(baseURL, urls...)
+	return urlsToResourceNames("/cluster/members", urls...)
```

Comment:
Would that also work?
```suggestion
	return urlsToResourceNames(u.String(), urls...)
```

Reply:
Indeed!